src/app/BugReporter/BugReportForm.cs

```diff
@@ -77,6 +78,7 @@ public BugReportForm()
 
             // ToDo: Displaying report contents properly requires some more work.
             mainTabs.TabPages.Remove(mainTabs.TabPages["reportContentsTabPage"]);
+            Load += (s, e) => ((Form)s!).FixVisualStyle();
```

Comment:
```suggestion
            Load += (s, e) => this.FixVisualStyle();
```

Generally, it's better to override `OnLoad` as that avoid allocations for the event and the eventhandler.

Reply:
changed, also for the existing call